docs/howto/deployment/wsgi/modwsgi.txt

```diff
@@ -103,11 +105,25 @@ Django instance to run in it, you will need to add appropriate
 ``WSGIDaemonProcess`` and ``WSGIProcessGroup`` directives. A further change
 required to the above configuration if you use daemon mode is that you can't
 use ``WSGIPythonPath``; instead you should use the ``python-path`` option to
-``WSGIDaemonProcess``, for example::
+``WSGIDaemonProcess``, for example:
+
+.. code-block:: apache
 
     WSGIDaemonProcess example.com python-path=/path/to/mysite.com:/path/to/venv/lib/python2.7/site-packages
     WSGIProcessGroup example.com
 
+Additionally, daemon mode can be used to specify a non root subdirectory of
```

Comment:
Technically non-daemon mode can be used for this too, but daemon is recommended in general (assuming you're not on Windows :)